### ISSUE TITLE
Making a special endpoint to return a collection of Entities

### ISSUE THREAD

**Issue by alexglvr** [api-platform/api-platform#1750]
Hello,

In my application, I have several entities configured via admin interface.
My Main Object (MO), has linked to all these entities.

Therefore, opening the MO creation form in my client results in plenty of API requests to load all the entities.
Most of browsers are now limited to a maximum number of simutaneous request to the same IP. Also, the loading is slown down, several requests beeing blocked the time the previous ones are finished.

Is there a way to create a special endpoint which will return a collection of all the mandatory entities?
This would permit to sumarize all the requests in a single one...

if not, is there any chance this functionality woud be added in future release?

Thank you for your help,

Alex

**Comment by rugolinifr** [comment]
> Most of browsers are now limited to a maximum number of simutaneous request to the same IP.

No, it is the number of simultaneous **connections**. This problem is solved using HTTP 2, which [re-use the same connection](https://http2-explained.haxx.se/en/part6#6-3-multiplexed-streams) for each request to the same host.

For example, take a look at Google Image, which serves using HTTP 2, you'll see that every image is downloaded fast, at the same moment, with 0ms of [**blocked**](https://developer.mozilla.org/en-US/docs/Tools/Network_Monitor/request_details#Request_Timing) time.

**Comment by soyuka** [comment]
Closing per #2987. Questions are best asked on [GitHub Discussions](https://github.com/api-platform/core/discussions) or [Stack Overflow (api-platform tag)](https://stackoverflow.com/questions/tagged/api-platform.com). The framework now lives at `api-platform/core`.
